pkg/config/config.go

```diff
@@ -65,6 +65,20 @@ var (
 	LogMaxAge = ffc("log.maxAge")
 	// LogCompress sets whether to compress backups
 	LogCompress = ffc("log.compress")
+	// LogReportCaller enables the report caller for including the calling file and line number
+	LogReportCaller = ffc("log.reportCaller")
```

Comment:
Noting that the text formatter we use doesn't currently respect this:

```yaml
log:
  level: debug
  reportCaller: true
  noColor: true
```

```
[2022-04-20T05:04:35.099Z] DEBUG node_0: Log level: debug
[2022-04-20T05:04:35.099Z]  INFO node_0: Project Firefly
[2022-04-20T05:04:35.099Z]  INFO node_0: © Copyright 2021 Kaleido, Inc.
[2022-04-20T05:04:35.100Z] DEBUG node_0: Debug HTTP endpoint listening on localhost:6060
```

so could use input on whether this option should be fall under `json` or not.

Reply:
See note above on spelling

Reply:
Does the other logger formatter support it?
e.g. if I set `reportCaller` (or `includeCodeInfo` etc.) do my _non_ JSON logs include it?

Reply:
Not currently no, its unclear if the `prefixed` text formatter we use doesn't support logrus' report caller or if maybe because we're setting `ForceFormatting`. I'll play with it a bit

Reply:
If it only works for `json`... then best to just pop it under the `json` section I think.

Reply:
In theory it shouldn't only work for `json`, so gonna try to understand why that's not working before giving up and moving it under there. Could definitely see local stacks benefitting from have `includeCodeInfo` enabled with human-readable text logs.